README.md

```diff
@@ -2,14 +2,15 @@
   <a href="https://svelte.dev">
 	<img alt="Cybernetically enhanced web apps: Svelte" src="https://sveltejs.github.io/assets/banner.png">
   </a>
-
   <a href="https://www.npmjs.com/package/svelte">
     <img src="https://img.shields.io/npm/v/svelte.svg" alt="npm version">
   </a>
-
   <a href="https://github.com/sveltejs/svelte/blob/master/LICENSE">
     <img src="https://img.shields.io/npm/l/svelte.svg" alt="license">
   </a>
+	<a href="https://svelte.dev/chat">
+		<img src="https://img.shields.io/badge/chat-on%20discord-7289da.svg?sanitize=true" alt="Chat">
```

Comment:
What does `?sanitize=true` do? We don't have that on other shields.io images.

Reply:
100% no difference; removing it. Plus there's a spacing inconsistency, so fixing that too...